src/configuration/services/network-storage.md

```diff
@@ -0,0 +1,170 @@
+
+```yaml
+name: app
+
+type: php:7.2
+
+disk: 1024
+
+mounts:
+    'network_dir':
+        source: service
+        service: files
+        source_path: our_stuff
+
+    'local_dir':
+        source: local
+        source_path: my_stuff
+
+web:
+    locations:
+        "/":
+            root: "public"
+            passthru: "/index.php"
+
+workers:
+    queue:
+        commands:
+            start: |
+                php worker.php
+```
+
+In this case, both the web instance and the `queue` worker will have two mount points: `network_dir` and `local_dir`.  
+
+* The `local_dir` mount on each will be independent and not connected to each other at all, and they will *each* take 1024 MB of space.
+* The `network_dir` mount on each will point to the same network storage space on the `files` service.  They will both be able to read and write to it simultaneously.  The amount of space it has available will depend on the `disk` key specified in `services.yaml`.
+
+## How do I give my workers access to my main application's files?
+
+The most common use case for `network-storage` is to allow a CMS-driven site to use a worker that has access to the same file mounts as the web-serving application.  For that case, all that is needed is to set the necessary file mounts as `service` mounts.
+
+For example, the following `.platform.app.yaml` file (fragment) will keep a Drupal files directories shared between web and worker instances while keeping the Drush backup directory web-only (as it has no need to be shared).  (This assumes a service named `files` has already been defined in `services.yaml`.)
+
+
+```yaml
+name: 'app'
+type: 'php:7.2'
+
+relationships:
+    database: 'mysqldb:mysql'
+
+hooks:
+  # ...
+
+web:
+    locations:
+        '/':
+            # ...
+
+disk: 1024
+
+mounts:
+    # The public and private files directories are network mounts shared by web and workers.
+    'web/sites/default/files':
+        source: service
+        service: files
+        source_path: files
+    'private':
+        source: service
+        service: files
+        source_path: private
+    # The backup, temp, and cache directories for Drupal's CLI tools don't need to be shared.
+    # It wouldn't hurt anything to make them network shares, however.
+    '/.drush':
+        source: local
+        source_path: drush
+    'tmp':
+        source: local
+        source_path: tmp
+    'drush-backups':
+        source: local
+        source_path: drush-backups
+    '/.console':
+        source: local
+        source_path: console
+
+# Crons run on the web container, so they have the same mounts as the web container.
+crons:
+    drupal:
+        spec: '*/20 * * * *'
+        cmd: 'cd web ; drush core-cron'
+
+# The worker defined here will also have 6 mounts; 2 of them will be shared with the web container, the other 4 will be local to the worker.
```

Comment:
Note that the lines aren't all rendered depending on the screen size. Consider limiting the amount of char for each line to a lower number (80, maybe?). 

How it's now, is very difficult to read the comments: I need to scroll down until the last line from this `yaml` file, then scroll right, then scroll up and continue reading where I've stopped.